packages/html/src/mediaplayer/mediaplayer-toolbar.tsx

```diff
@@ -0,0 +1,74 @@
+import { KendoToolbarProps, Toolbar } from '../toolbar';
+import { classNames } from '../misc';
+import { Button } from '../button';
+import { DropdownList } from '../dropdownlist';
+import { SliderNormal } from '../slider';
+
+export const MEDIAPLAYERTOOLBAR_CLASSNAME = `k-mediaplayer-toolbar`;
+
+const states = [];
+
+const options = {};
+
+export type KendoMediaPlayerToolbarProps = {
+    currentTime?: string;
+    durationTime?: string;
+    showQuality?: boolean;
+    qualityValue?: string;
+};
+
+const defaultProps = {
+    currentTime: "00:18",
+    durationTime: "00:55",
+    showQuality: true,
+    qualityValue: "360p"
+};
+
+export const MediaPlayerToolbar = (
+    props: KendoToolbarProps &
+        KendoMediaPlayerToolbarProps &
+        React.HTMLAttributes<HTMLDivElement>
+) => {
+    const {
+        currentTime = defaultProps.currentTime,
+        durationTime = defaultProps.durationTime,
+        showQuality = defaultProps.showQuality,
+        qualityValue = defaultProps.qualityValue,
+        ...other
+    } = props;
+
+    return (
+        <Toolbar
+            {...other}
+            className={classNames(
+                props.className,
+                MEDIAPLAYERTOOLBAR_CLASSNAME,
+            )}
+        >
+            <Button fillMode="flat" icon="play" className="k-play-button"></Button>
+            <div className="k-mediaplayer-currenttime-wrap">
+                <span className="k-mediaplayer-currenttime">{currentTime}</span>
+                /
+                <span className="k-mediaplayer-duration">{durationTime}</span>
+            </div>
+            <div className="k-spacer"></div>
+            <Button fillMode="flat" icon="volume-up" className="k-volume-button"></Button>
+            <div className="k-mediaplayer-volume-wrap">
+                <SliderNormal className="k-mediaplayer-volume" showButtons={false} showTicks={false} />
+            </div>
+            {showQuality &&
+                <div className="k-mediaplayer-quality-wrap">
+                    <DropdownList value={qualityValue} arrowIconName="hd" className="k-mediaplayer-quality" />
+                </div>
+            }
+            <Button fillMode="flat" icon="fullscreen" className="k-fullscreen-button"></Button>
```

Comment:
We should fully re-use the Toolbar here. The template items( volume, quality, etc. ) should be rendered inside .k-toolbar-item. See [ref](https://demos.telerik.com/kendo-ui/mediaplayer/index) and [ref2](https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/toolbar/tests/toolbar.tsx#L53).